experiments/experiment.go

```diff
@@ -82,11 +82,12 @@ func (ec *experimentContext) reconcile() *v1alpha1.ExperimentStatus {
 		ec.reconcileAnalysisRun(analysis)
 	}
 
-	if duration := calculateEnqueueDuration(ec.ex); duration != nil {
```

Comment:
Oh, we were only acting on the previous experiment... Yeah that would cause problems